addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -879,10 +894,9 @@ var SnippetEditor = Widget.extend({
             $selectorChildren: $selectorChildren,
         });
 
-        this.$body.addClass('move-important');
```

Comment:
Hello, I removed it because the _getScrollOptions are using the 'grabbing' cursor by default. Weirdly when I test it does not work on firefox when moving pictures, the cursor is displayed instead of the closed hand. 
I don't understand why the cursor has to be forced in the start/stop methods, maybe it is changed in the css somewhere, but reactivating those lines removes the bug for dragging the pictures. Maybe the class should be changed to .grabbing-important to be consistent with the jquery-ui options ? I changed from move to grabbing because the safari 'move' is not a closed hand